mandatory/1-fix-functions.js

```diff
@@ -21,7 +20,8 @@ function mood() {
 }
 
 function greaterThan10(num) {
-  let isBigEnough;
+  let isBigEnough
+   isBigEnough = num>10;
```

Comment:
Hi please make sure you have Prettier installed and that it is formatting your code. 

https://syllabus.codeyourfuture.io/guides/code-style-guide#using-prettier-to-format-code-automatically

Reply:
to be honest i had installed but i hadn't set default formatter. now it is done.